Assets/Scripts/Model/AbstractPlayer.cs

```diff
@@ -7,7 +8,11 @@ public abstract class AbstractPlayer : DynamicGameObject
         protected int _remainingLives;
         public int remainingLives {
             get { return _remainingLives; }
-            set { _remainingLives = value; OnLivesChanged(value); }
+            set { if (value <= Constants.MAX_LIVES_COUNT)
+                {
+                    _remainingLives = value; OnLivesChanged(value);
```

Comment:
To nie jest odpowiedzialność tej klasy. Ograniczenie żyć jest spowodowane HUDem, ale nic złego się nie dzieje w momencie, gdy żyć mamy więcej niż jesteśmy w stanie wyświetlić. To GameManager powinien zwracać zerową ilość znajdziek dla poziomu, jeżeli gracz ma już MAX_LIVES_COUNT żyć.
- [ ] zostawić stałe w Constantsach tak jak są
- [ ] wyrzucić ograniczenia na ilość żyć w AbstractPlayerze